releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,15 @@
 # Microsoft.Extensions.Configuration.AzureAppConfiguration
 [Source code][source_code] | [Package (NuGet)][package]
 
+## 8.0.0-preview.3 - July 18th, 2024
+### Breaking Changes
+* Removed .NET 7 as a target framework as .NET 7 is out of support. [#567](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/567)
+* The provider now uses the [Microsoft feature flag schema v2.0.0](https://github.com/Azure/AppConfiguration/blob/main/docs/FeatureManagement/FeatureFlag.v2.0.0.schema.json) to output feature flags containing any of the `variants`, `allocation` or `telemetry` properties to configuration. Flags without these properties will not change in configuration. [#543](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/543)
```

Comment:
Is this a behavior change compared to the preview 2 release?

Reply:
It's only in what's output to configuration, I just assumed that in the case that someone is reading it directly instead of using the feature management library it would be a breaking change.

Reply:
Not an answer to my question :). I'm hesitant to call this a breaking change because we are making this very change for better compatibility.

Reply:
Sorry I missed this initially. I see, yea now that I think about it this can just go under enhancements.